package.json

```diff
@@ -1,56 +1,82 @@
 {
-  "name": "timekit-booking",
-  "version": "2.7.0",
-  "description": "Make a beautiful embeddable booking widget in minutes",
-  "main": "dist/booking.min.js",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/timekit-io/booking-js.git"
-  },
-  "scripts": {
-    "watch": "webpack --watch & webpack --config webpack.config.min.js --watch && kill $!",
-    "build": "webpack && webpack --config webpack.config.min.js --bail",
-    "release:patch": "../flux/flux timekit:release patch booking-js",
-    "release:minor": "../flux/flux timekit:release minor booking-js",
-    "release:major": "../flux/flux timekit:release major booking-js",
-    "deploy:prod": "../flux/flux timekit:deploy prod booking-js",
-    "deploy:staging": "../flux/flux timekit:deploy staging booking-js",
-    "test": "webpack && webpack --config webpack.config.min.js && karma start --single-run",
-    "test:watch": "webpack && webpack --config webpack.config.min.js && karma start --watch"
-  },
-  "author": "Timekit Inc.",
-  "license": "MIT",
-  "dependencies": {
-    "console-polyfill": "^0.2.2",
-    "es6-promise-promise": "^1.0.0",
-    "fullcalendar": "^3.4.0",
-    "json-stringify-safe": "^5.0.1",
-    "moment": "^2.10.6",
-    "moment-timezone": "^0.5.13",
-    "querystringify": "^2.0.0",
-    "sprintf-js": "^1.0.3",
-    "timekit-sdk": "^1.16.0"
-  },
-  "devDependencies": {
-    "autoprefixer-loader": "^3.1.0",
-    "css-loader": "^0.21.0",
-    "jasmine": "^2.3.2",
-    "jasmine-jquery": "^2.1.1",
-    "jquery": "^3.2.1",
-    "json-loader": "^0.5.4",
-    "karma": "^0.13.0",
-    "karma-chrome-launcher": "^0.1.12",
-    "karma-jasmine": "^0.3.5",
-    "karma-jasmine-ajax": "^0.1.12",
-    "karma-jasmine-jquery": "^0.1.1",
-    "karma-sourcemap-loader": "^0.3.5",
-    "karma-spec-reporter": "0.0.19",
-    "karma-webpack": "^1.7.0",
-    "mustache-loader": "https://github.com/laander/mustache-loader#loader-utils-as-dependency",
-    "node-sass": "^4.13.1",
-    "sass-loader": "^3.1.1",
-    "style-loader": "^0.13.0",
-    "svg-inline-loader": "^0.3.0",
-    "webpack": "^1.12.2"
-  }
+	"version": "2.7.1",
+	"name": "timekit-booking",
+	"description": "Make a beautiful embeddable booking widget in minutes",
+	"main": "dist/booking.min.js",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/timekit-io/booking-js.git"
+	},
+	"scripts": {
+		"watch": "webpack --watch & webpack --config webpack.config.min.js --watch && kill $!",
+		"build": "webpack && webpack --config webpack.config.min.js",
```

Comment:
Now that we're not using bail, do we need another safe guard in place?

Reply:
added bail option to webpack.conf.min file